deploy-manage/security/private-link-traffic-filters.md

```diff
@@ -16,8 +18,14 @@ Choose the relevant option for your cloud service provider:
 | Azure | [Azure Private Link](/deploy-manage/security/azure-private-link-traffic-filters.md) |
 | GCP | [GCP Private Service Connect](/deploy-manage/security/gcp-private-service-connect-traffic-filters.md) |
 
-After you set up your private link, you can [claim ownership of your filter link ID](/deploy-manage/security/claim-traffic-filter-link-id-ownership-through-api.md) to prevent other organizations from using it in a traffic filter ruleset.
+After you set up your private connection, you can [claim ownership of your VCPE ID](/deploy-manage/security/claim-traffic-filter-link-id-ownership-through-api.md) to prevent other organizations from using it.
```

Comment:
```suggestion
After you set up your private connection, you can [claim ownership of your VCP ID](/deploy-manage/security/claim-traffic-filter-link-id-ownership-through-api.md) to prevent other organizations from using it.
```